package.json

```diff
@@ -24,6 +24,7 @@
         "@angular/platform-browser": "^15.1.3",
         "@angular/platform-browser-dynamic": "^15.1.3",
         "@angular/router": "^15.1.3",
+        "@material/dialog": "^15.0.0-canary.684e33d25.0",
```

Comment:
is this necessary? Dialog comes with angular material package I think!

Reply:
https://blog.angular-university.io/angular-material-dialog/
what do you think about this

Reply:
what about that? I am asking do we need this: `"@material/dialog": "^15.0.0-canary.684e33d25.0",`

Reply:
Yes but i think how As it is meant to be, so be it